bin/heudiconv

```diff
@@ -58,9 +95,64 @@ SeqInfo = namedtuple(
      'referring_physician_name',
      'series_description',
      'image_type',
+     'accession_number',
+     'patient_age',
+     'patient_sex',
+     'date'
      ]
 )
 
+StudySessionInfo = namedtuple(
+    'StudySessionInfo',
+    [
+        'locator',  # possible prefix identifying the study, e.g. PI/dataset or just a dataset or empty (default)
+                    # Note that ATM there should be no multiple DICOMs with the same
+                    # StudyInstanceUID which would collide, i.e point to the same
+                    # subject/session.
+                    # So 'locator' is pretty much an assignment from StudyInstanceUID
+                    # into some place within hierarchy
+        'session',  # could be None
+        'subject',  # should be some ID defined either in cmdline or deduced
+    ]
+)
+
+
+class TempDirs(object):
```

Comment:
Valid concern!  But primarily for the case whenever such refactoring would (and if, since original idea was to have a single script IIRC) happen -- ATM it is just as fine imho, so just added a TODO comment on that